sdk/core/core-http/test/serviceClientTests.ts

```diff
@@ -44,7 +44,7 @@ describe("ServiceClient", function() {
 
     const testOperationSpec: OperationSpec = {
       httpMethod: "GET",
-      baseUrl: "httpbin.org",
+      baseUrl: "https.bin.org",
```

Comment:
This is bad, right? Ie, it really should only match against https://, not just a random https starting string.

Reply:
oh good point!